### PR TITLE
HaplotypeCaller now creates gVCF - localhost settings optimized for irma

### PR DESCRIPTION
**Pull request by szilvajuhos**

### CODE REVIEW

**Comment by MaxUlysse** [comment]
should we name `localhost.config` to `irma.config` ?

**Comment by szilvajuhos** [comment]
Not really, milou/bianca compute nodes are similar.

File: main.nf [hunk 1]
```diff
@@ -775,7 +776,8 @@ process ConcatVCF {
 	done
 	cat header raw_calls > unsorted.vcf
 	java -jar \${PICARD_HOME}/picard.jar SortVcf I=unsorted.vcf O=$outputFile
-    rm unsorted.vcf
+	rm unsorted.vcf
+	gzip -v $outputFile
```
Comment (marcelm) [inline main.nf]: Can you pipe into gzip to avoid some I/O? I think picard accepts `O=/dev/stdout`

Comment (szilvajuhos) [inline main.nf]: Looks not this one:
`$ java -jar ${PICARD_HOME}/picard.jar SortVcf I=bugger.vcf OUTPUT=/dev/stdout 
[Wed Mar 22 14:42:27 CET 2017] picard.vcf.SortVcf INPUT=[bugger.vcf] OUTPUT=/dev/stdout    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT 
`
...


`Exception in thread "main" java.lang.IllegalArgumentException: Must specify file or stream output type.
        at htsjdk.variant.variantcontext.writer.VariantContextWriterBuilder.build(VariantContextWriterBuilder.java:407)
`

Comment (marcelm) [inline main.nf]: Ok, thanks for checking!

**Pull request merged**